src/Sulu/Bundle/AdminBundle/Resources/js/components/Modal/Actions.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+import React from 'react';
+import type {Action} from './types';
+import actionsStyles from './actions.scss';
+
+type Props = {
+    actions: Array<Action>,
+};
+
+export default class Actions extends React.PureComponent<Props> {
+    render() {
+        if (this.props.actions.length > 0) {
```

Comment:
use destructuring? At first I thought it doesn't make any sense, but you are using it twice, so it does for me.